store/root/store.go

```diff
@@ -87,17 +93,28 @@ func (s *Store) Close() (err error) {
 	s.lastCommitInfo = nil
 	s.commitHeader = nil
 
+	s.pruningManager.Stop()
+
 	return err
 }
 
+// SetPruningOptions sets the pruning options on the SS and SC backends.
+// NOTE: It will also start the pruning manager.
+func (s *Store) SetPruningOptions(ssOpts, scOpts pruning.Options) {
+	s.pruningManager.SetStorageOptions(ssOpts)
+	s.pruningManager.SetCommitmentOptions(scOpts)
+
+	s.pruningManager.Start()
```

Comment:
It is pretty important that s.pruningManager.Start have a sync.Once variable to catch multiple surprising calls to .Start()

Reply:
`Start` is idempotent, since we have `isStarted` field